resources/event-bus/charts/event-publish-service/templates/deployment.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
```

Comment:
May be, we can move these changes to another PR(will be created anyway) which will have the bumped image of event-bus.